test/clj/chronograph/db/users_test.clj

```diff
@@ -0,0 +1,47 @@
+(ns chronograph.db.users-test
+  (:require [clojure.test :refer :all]
+            [chronograph.db.users :as users-db]
+            [chronograph.fixtures :as fixtures]))
+
+(use-fixtures :once fixtures/config fixtures/datasource)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest create-and-retrieve-google-user-test
+  (testing "Can retrieve a user created by find-or-create"
+    (let [created-user   (users-db/find-or-create-google-user! "google-123"
```

Comment:
What do you think about creating a simple factory function, that takes a table and a spec and inserts some generated values into the table?

Reply:
This was intended as a test for `find-or-create` as well.
Also, it inserts into three different tables. So in this case, I think that using this function is more convenient anyway.